packages/playwright-config-commons-ui/next.js

```diff
@@ -0,0 +1,11 @@
+const defaultConfig = require(".");
```

Comment:
I don’t think this should be here.

Reply:
@kilemensi I think it should be there otherwise playwright + jest complain when you run `pnpm test`.

Reply:
I mean this next.js file means: here are the _Next.js_ specific configs for _Playwright_. Jest has specific requirements but Playwright doesn't (at least looking a [docs](https://nextjs.org/docs/testing#playwright)).

So we shouldn't need this file @nerdlet